src/main/java/emissary/util/PayloadUtil.java

```diff
@@ -61,22 +90,63 @@ public static String getPayloadDisplayString(final IBaseDataObject payload) {
         final StringBuilder sb = new StringBuilder();
         final List<TransformHistory.History> th = payload.getTransformHistory().getHistory();
         final String fileName = payload.getFilename();
+        final String fileType = payload.getFileType();
         final List<String> currentForms = payload.getAllCurrentForms();
         final Date creationTimestamp = payload.getCreationTimestamp();
 
         sb.append("\n").append("filename: ").append(fileName).append("\n").append("   creationTimestamp: ").append(creationTimestamp).append("\n")
-                .append("   currentForms: ").append(currentForms).append("\n").append("   filetype: ").append(payload.getFileType()).append("\n")
+                .append("   currentForms: ").append(currentForms).append("\n").append("   filetype: ").append(fileType).append("\n")
                 .append("   transform history (").append(th.size()).append(") :").append("\n");
-        for (final TransformHistory.History h : th) {
-            sb.append("     ");
-            if (h.wasCoordinated()) {
-                sb.append("  ");
-            }
-            sb.append(h.getKey()).append("\n");
+
+        // transform history output
+        String historyCase = configureHistoryCase(fileType);
+        switch (historyCase) { // leaving as switch for future transform history altercations
```

Comment:
this can be simplified to an if statement:
`if (REDUCED_HISTORY.equals(historyCase)) {`